packages/web-service/src/pages/habitat/name/habitat-name.js

```diff
@@ -1,6 +1,22 @@
+import Joi from 'joi'
 import pageRoute from '../../../routes/page-route.js'
 import { habitatURIs } from '../../../uris.js'
 
 export const completion = async _request => habitatURIs.TYPES.uri
 
-export default pageRoute({ page: habitatURIs.NAME.page, uri: habitatURIs.NAME.uri, completion })
+export const validator = async payload => {
```

Comment:
No need - look at the example of the contact name `packages/web-service/src/pages/contact/common/contact-name/contact-name-page.js`